src/stepFunctions/asl/client.ts

```diff
@@ -135,13 +136,13 @@ export async function activate(extensionContext: ExtensionContext) {
                     const params: DocumentRangeFormattingParams = {
                         textDocument: client.code2ProtocolConverter.asTextDocumentIdentifier(document),
                         range: client.code2ProtocolConverter.asRange(range),
-                        options: client.code2ProtocolConverter.asFormattingOptions(options),
+                        options: client.code2ProtocolConverter.asFormattingOptions(options, {}),
                     }
 
                     return client.sendRequest(DocumentRangeFormattingRequest.type, params, token).then(
                         response => client.protocol2CodeConverter.asTextEdits(response),
                         async error => {
-                            client.logFailedRequest(DocumentRangeFormattingRequest.type, error)
+                            getLogger().verbose('[step functions]: client request failed: %O', error)
```

Comment:
`logFailedRequest` method does not exist at all anymore. Presumably it just logged the message somewhere, so I replaced it with our own logger.